SalesforceCommerceCloud/pwa-kit
# [V2]add eslintrc.js to project templates

@W-10667147@


# Description
ESLint errors are no longer being picked up by editors due to the rc files living in the developer tools now and not directly in the project.

This PR adds the rc files into project templates




# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- pull the code
- Verify that your IDE could pick up the eslint setting in each template
- run `npm run lint` at root and verify that every thing works
# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
